src/foam/demos/u2/AllViews.js

```diff
@@ -479,7 +479,7 @@ foam.CLASS({
       class: 'FObjectArray',
       name: 'fobjectArray',
       of: 'foam.demos.u2.SampleData',
-      view: { class: 'foam.u2.view.FObjectArrayView', valueView: 'foam.demos.heroes.CitationView' }
+      view: { class: 'foam.u2.view.TitledArrayView', valueView: 'foam.demos.heroes.CitationView' }
```

Comment:
Since both are views, I'd not suggest removing the FObjectArrayView.
It would actually be fun to see them both. Id recommend using a multiView with both these arrayViews.

Reply:
Would it be better to use a multiview or add a different section for FObjectArray view? I feel like these views are meant to be used as fullwidth views anyway o adding them in different sections makes more sense

Reply:
@kgrgreer ^